docs/UserGuide.md

```diff
@@ -78,92 +77,149 @@ HobbyList is a **desktop app for managing hobby activities, optimized for use vi
-Show a list of all hobby activities whose rate is exactly the required value(between 0-5, both inclusive).
+
 Format:
-    `find KEYWORD`
-    `find rate/[VALUE]`
-    `find yyyy-mm-dd`
-    `find yyyy-mm`
-    `find yyyy`
+    `find KEYWORDS`
 
-* The keywords are case-insensitive.
+* The `KEYWORDS` are case-insensitive.
 
 Example:
 * `find run`
 * `find mystery novel`
+
+#### With time:
+
+Shows a list of all hobby activities that located on a certain date, or in a certain month of a year, or in a certain year.
+
+Format:
+* `find YYYY-MM-DD`
+* `find YYYY-MM`
+* `find YYYY`
+
+Example:
+* `find 2001-01-01`
 * `find 1974-02`
 * `find 1999`
-* `find 2001-01-01`
-* `find rate/0`
 
-### Finding activities with rate above certain value : `r/above value`
+#### With exact rating value:
 
-Show a list of all hobby activities whose rate value is greater or equal than the required value.
-Format:
-    `r/above [VALUE]`
+Shows a list of all hobby activities whose rating is exactly the required value(between 0-5, both inclusive).
+
+Format: `find rate/VALUE`
+
+* `VALUE` only accepts integer numbers from 0 to 5 (inclusive).
+
+Example: `find rate/3`
+
+### Finding activities with rating value above certain value : `r/above`
 
-* The value need to be between 0-5(inclusive).
+Shows a list of all hobby activities whose rating value is greater or equal than the required value.
 
-Example"
-* `r/above 1`
+Format: `r/above VALUE`
 
-### Finding activities by tags : `filter`
+* `VALUE` only accepts integer numbers from 0 to 5 (inclusive).
+
+Example: `r/above 1`
+
+### Finding activities with tags : `filterTag`
 
 Shows a list of all hobby activities whose tags match the one specified in the command.
 
-Format: `filter KEYWORD`
+Format: `filterTag KEYWORDS`
 
-* The keywords are case-insensitive.
+* The `KEYWORDS` are case-insensitive.
 
 Example:
-* `filter book`
-* `filter anime`
+* `filterTag book`
+* `filterTag ANIME`
+
+### Finding activities with status: `filterStatus`
+
+Shows a list of all hobby activities whose status match the status specified in the command.
+
+Format: `filterStatus STATUS`
+
+* The feature only works with the `STATUS` being **UPCOMING**, **ONGOING** or **COMPLETED** (case-insensitive).
+* If `STATUS` is different from the three statuses above, it is treated as **NONE**.
+
+Example: 
+* `filterStatus COMPLETED`
+* `filterStatus ongoing`
 
 ### Editing an activity : `edit`
 
 Edits a specified activity from HobbyList.
 
 Format: `edit INDEX [n/NAME] [d/DESCRIPTION] [t/TAGS]... [date/DATE] [s/STATUS]`
 
-* Edits the person at the specified INDEX. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …
-* At least one of the optional fields must be provided.
+* Edits the activity at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The `INDEX` **must be a positive integer** 1, 2, 3, …
```

Comment:
Small mistake, should be "displayed activity list" instead of "displayed person list"